templates/operations.ts.hbs

```diff
@@ -159,6 +159,8 @@
       Object.keys(optionParams).forEach((key) => {
         if(key.startsWith('c_') && optionParams[key as keyof typeof optionParams] !== undefined) {
           queryParams[key as keyof typeof queryParams] = optionParams[key as keyof typeof optionParams]
+        } else if(!queryParams.hasOwnProperty(key) && !pathParams.hasOwnProperty(key)) {
+          throw new Error(`Invalid Parameter: ${key}`)
```

Comment:
This would be considered a breaking change. Do we need to be doing this? 
Also, it wouldn't be in alignment with the comment in the docs of this other pr [here](https://github.com/SalesforceCommerceCloud/commerce-sdk/pull/398/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R230)